pkg/resources/validation/clusterroletemplatebinding/clusterrtb.go

```diff
@@ -31,12 +35,28 @@ func (c *clusterRoleTemplateBindingValidator) Admit(response *webhook.Response,
 	if err != nil {
 		return err
 	}
-
-	if crtb.ClusterName != "local" {
+	if crtb.ClusterName != "local" { // the prtb custom resource only exists in the local cluster
```

Comment:
Two Things here:

1. I don't think that the ClusterName in this case refers to the name where the object is in. I think this is the name of the cluster that the binding is in effect for (e.x. user y has access to role rt in cluster x -> in this case, x is the ClusterName). Do we really want to have a short circut in here? I kind of think that this should be removed.
2. Nit: this file is for crtbs, but the comment references prtbs. I recommend revising this.